projects/quests.md

```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
 
+## Pages
+
+### List view
+
+![](https://i.imgur.com/hRtXi66.png)
+
+This page consists of a small dashboard and a list of summarized Quests.
+
+The dasboard shows the following information:
+
+- Total of the active Quests bounties in USD
+- Number of active Quests
+- A **Create quest** button
+
+This list can be filtered by Quest title, description, expiration date and state.
+
+The expired Quests can be accessed by selecting **Expired** in the state filter.
+
+### Detail view
+
+This page is the detailed version of the quest showing more information :
+
+- The claim deposit (amount locked during the claiming process)
+- The full description (not summarized)
+- The list of the **Players**' claims
+- The action buttons (needs to be connected)
+  - Fund (where Patrons can fund the quest)
+  - Schedule claim (where Players can submit their evidence of task completion)
+
+![](https://i.imgur.com/HlJA7fs.png)
+
+## Create a quest
+
+**Creators** can create a Quest only when connected to a wallet. The **Create quest** buttons will then appear in the header/dashboard/footer.
+
+The creator must submit the following in 2 steps :
+
+1. First:
+   - Title
+   - [Description](quests.md#quest-descriptions) (the field supports markdown)
+2. Second:
+   - Funding token (any token address can be put in the field)
+   - Initial bounty
+   - Expire time (after that, Players will not be able to schedule new claims)
+   - Fallback address (the address where the funds can be restored once the quest expires)
+3. The Creator needs to deposit a collateral that will be locked until the Quest expiration date. He will need to Reclaim the deposit to recover it
+
+## Fund a quest
+
+**Patrons** can fund a Quest only when connected to a wallet. The Fund button is in the detail view of a Quest.
+
+The Patron should only submit the amount of funding for the Quest reward token.
+
+## Claim a Quest bounty
+
+**Players** can claim the bounty only when connected to a wallet. The **Schedule claim** button is in the detailed view of a Quest.
+
+The submitted claim will be delayed for a specific period where other users can challenge the claim if the requirements of the Quest aren't met (see [verification of Quest](quests.md#verifying-a-quest)).
+
+The Player msut submit the following in 2 steps :
+
+1. First:
+   - Evidence of completion (should meet the Quest description requirements)
+2. Second:
+   - The UI offers to claim all the Quest's bounty or just a specific amount (Player should pick what is specified in the Quest description)
+   - Player's wallet address to which the reward should be sent when a claim is completed
+3. The Player should deposit a collateral that will be restored once the claim is executed successfully.
```

Comment:
If not successfully, what happen?

Reply:
Its explain somewhere else but an unsuccessful claim is an accepted challenge but yeah I think its more confusing, I'll remove the susscesffully part